tmpfiles.d/systemd.conf.m4

```diff
@@ -7,6 +7,8 @@
 
 # See tmpfiles.d(5) for details
 
+d /run/environment.d 0755 root root -
```

Comment:
why this? if you want to be able to put a file there you need privs anyway, and can hence create the dir... an there's nothing too special in the access mode tha twould prohibit that...

I don't think this file should exist, except when people actually need this...